src/fetchAPI.js

```diff
@@ -1,24 +1,26 @@
+const key = '2ee2ed8e-4950-4027-9dfb-a2c0e35f3218'
```

Comment:
Good call here.

Reply:
Definitely. thanks for replacing that key - maybe we can see if there's a better way to deal with this than just replacing the key every so often, but it seems to have done the trick for now.